config/locales/controllers/lti/en.yml

```diff
@@ -6,5 +6,4 @@ en:
       notice_students_open_in_cv: "Please open the Assigment in CircuitVerse"
       notice_ask_teacher: "Please ask your teacher to add you in the CircuitVerse group associated with this course."
       notice_no_account_in_cv: "You have no account associated with email %{email_from_lms} please create first and try again."
-      notice_invalid_group: "There is no group in CircuitVerse associated with your current LMS, Please ask your LMS Admin/Teacher to create one"
-    
+      notice_invalid_group: "There is no group in CircuitVerse associated with your current LMS, Please ask your LMS Admin/Teacher to create one"
```

Comment:
delete this line also `notice_invalid_group` and keep the file unchanged like previous version